html/projects.html

```diff
@@ -0,0 +1,202 @@
+                    {'field': 'leader.name.exact', 'size': 20, 'display': 'Leader'},
+                    {'field': 'participants.name.exact', 'size': 20, 'display': 'Participants'},
+                    {'field': 'submittedBy.name.exact', 'size': 20, 'display': 'Submitted By'},
+                    {'field': 'grant.label.exact', 'size': 20, 'display': 'Grants'},
+                    {'field': 'dateTimeInterval.startYear', 'size': 20, 'display': 'Start Year'},
+                    {'field': 'dateTimeInterval.endYear', 'size': 20, 'display': 'End Year'},
+                    {'field': 'reportingYear.year', 'size': 20, 'display': 'Reporting Year'}
+                ],
+                search_sortby: [
+                    {'display':'Title','field':'title.exact'}
+                ],
+                render_result_record: function(options, record)
+                {
+
+                    var html = "<tr><td>";
+
+                    if (record["thumbnail"]) {
+                        html += "<div class='thumbnail'>"
+                        html += "<img src='"+record["thumbnail"]+"'>&nbsp;";
+                        html += "</div>"
+                    }
+
+                    html += "<div class='project-info'>"
+
+                    if (record["uri"]) {
+                        html += "<strong><a href=\""+record["uri"]+"\" target=\"_blank\">" + record["title"] + "</a></strong>";
+                    } else {
+                        html += "<strong>" + record["title"] + "</strong>";
+                    }
+
+                    if (record["mostSpecificType"]) {
+                        html += "<br />Type: " + record["mostSpecificType"];
+                    }
+
+                    if (record["leader"]) {
+                        html += "<br /><span>Leader: <a href=\"" + record["leader"]["uri"] + "\" target=\"_blank\">" + record["leader"]["name"] + "</a></span>";
+                    }
+
+                    if (record["community"]) {
+                      if (record["community"].length > 1) {
+                        html += "<br /><span><small>Communities: ";
+
+                      } else {
+                        html += "<br /><span><small>Community: ";
+
+                      }
+                        for (var i = 0; i < record["community"].length; i++) {
+                            html += "<a href=\"" + record["community"][i]["uri"] + "\" target=\"_blank\">" + record["community"][i]["name"] + "</a>";
+                            if (i < record["community"].length - 1) {
+                                html += "; ";
+                            }
+                        }
+                        html += "</small></span>";
+                    }
+
+                    if (record["portalGroup"]) {
+                        html += "<br /><span>Portal Group: <a href=\"" + record["portalGroup"]["uri"] + "\" target=\"_blank\">" + record["portalGroup"]["name"] + "</a></span>";
+                    }
+
```

Comment:
This property still baffles me. I need to work this out with John. For now please do not display it. Comment it out for now.